cove/settings.py

```diff
@@ -86,6 +86,16 @@
                       'record': 'record-package-schema.json'},
         'default': None
     },
+    'schema_version': {  # Default schema version
+        'cove-ocds': '1.0.2',
+        'cove-360': None,
+        'default': None
+    },
+    'schema_version_choices': {
+        'cove-ocds': ('1.0.2', '1.0.1', '1.0.0'),  # Keep descending order when adding new versions
```

Comment:
Yes, this has been change in the latest code push to produce `'1.1-dev'` and `'1.0'` (using a slightly different approach for schema versions in setting.py)